packages/create-react-app/index.js

```diff
@@ -182,7 +182,25 @@ function run(root, appName, version, verbose, originalDirectory, template) {
 
   install(allDependencies, verbose, function(code, command, args) {
     if (code !== 0) {
-      console.error(chalk.cyan(command + ' ' + args.join(' ')) + ' failed');
+      console.log();
+      console.error('Aborting installation.', chalk.cyan(command + ' ' + args.join(' ')), 'has failed.');
+      // On 'exit' we will delete these files from target directory.
+      var knownGeneratedFiles = [
+        'package.json', 'npm-debug.log', 'yarn-error.log', 'node_modules'
```

Comment:
I believe globbing would add additional complexity, and/or an extra dep, so I want to make sure it's necessary before we do it.

@chitchu can you see how hard this would be?